packages/codemods/testUtils/index.ts

```diff
@@ -13,7 +13,7 @@ const formatCode = (code: string) => {
 export const matchTransformSnapshot = (
   transformName: string,
   fixtureName: string = transformName,
-  parser: 'ts' | 'tsx' | 'js' = 'tsx'
+  parser: 'ts' | 'tsx' | 'babel' = 'tsx'
```

Comment:
I don't think this is correct!

Reply:
I can't remember if we tried it on the CLI and found out otherwise but I was just going off the usage docs which say babel: 
![image](https://user-images.githubusercontent.com/32992335/135486897-de90fd5e-b87e-4d23-b62a-ec6a4e95aa6a.png)